apps/backend/.env.bu

```
@@ -0,0 +1,56 @@
+AUTH_GITHUB_ID=Ov23liVhiU5ZgR9b3Z97
+AUTH_GITHUB_SECRET=8ba0e1533eea00a5d09b7233e8b88ebf38dbb974
+
+AUTH_ORCID_ID=abcd12345
+AUTH_ORCID_SECRET=secret12345
+
+# Database configuration
+DATABASE_URL=postgresql://postgres:postgres@localhost:5432/openjii_local
+
+# Databricks configuration
+DATABRICKS_HOST=https://dbc-6efd58ae-21b6.cloud.databricks.com
+DATABRICKS_CLIENT_ID=5b328752-0dd6-4fdf-879e-aa4f910010c4
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABRICKS_CLIENT_ID key should go before the DATABRICKS_HOST key

(UnorderedKey)